platform/mds/mds-test-bundle/src/test/java/org/motechproject/mds/test/osgi/MdsDdeBundleIT.java

```diff
@@ -500,6 +543,90 @@ public void testCascadeDelete() {
     }
 
     @Test
+    public void testCascadeDeleteUniDirectional() {
+        ChildCase child1 = new ChildCase("Child 1");
+        childCaseDataService.create(child1);
+
+        ChildCase child2 = new ChildCase("Child 2");
+        childCaseDataService.create(child2);
```

Comment:
Side note - it shouldn't be required to manually create related objects. They should be by default created with the owning instance (mother).